apps/e2e-test/tsconfig.app.json

```diff
@@ -6,5 +6,5 @@
     "types": ["node"]
   },
   "exclude": ["jest.config.ts", "**/*.spec.ts", "**/*.test.ts"],
-  "include": ["**/*.ts"]
+  "include": ["**/*.ts", "src/steps/app-specific/workflow"]
```

Comment:
We should specify the extension name here instead of path

Reply:
Fixed.